t/mojo/roles.t

```diff
@@ -47,10 +47,10 @@ sub hello {'hello mojo!'}
 
 package main;
 
-use Mojo::ByteStream;
-use Mojo::Collection;
+use Mojo::ByteStream ();
+use Mojo::Collection ();
```

Comment:
Also wrong, no default export.

Reply:
The idea here is that even though the bare parens actually don't do anything, they serve to document that this statement will not be the source of any imported symbols. For someone already familiar with those classes it's obvious, but for someone else maybe not so much.

Having said that, I can force-push some changes with these removed. What I'm trying to do is get an idea of what people prefer when it comes to imports and this is helpful feedback.

Reply:
I don't think it matters in this case but doesn't calling use with bare parens prevent calling import? That would indeed change behavior in some cases

Reply:
Right. The bare parens can actually change the behaviour, since `import()` won't get called at all in that case.